UI/src/view/linear_asset/laneModellingForm.js

```diff
@@ -0,0 +1,597 @@
+        var fieldValues = [];
+        if (asset.properties) {
+          var existingProperty = _.find(asset.properties, function (property) {
+            return property.publicId === field.publicId;
+          });
+          if (!_.isUndefined(existingProperty))
+            fieldValues = existingProperty.values;
+        }
+        var dynamicField = _.find(self.dynamicFormFields, function (availableFieldType) {
+          return availableFieldType.name === field.type;
+        });
+        var fieldType = new dynamicField.fieldType(self._assetTypeConfiguration, field, isDisabled);
+        forms.addField(fieldType, sideCode);
+        var fieldElement = isReadOnly ? fieldType.viewModeRender(field, fieldValues) : fieldType.editModeRender(fieldValues, sideCode, setAsset, getValue);
+
+        var additionalElement = infoLabel(field.publicId);
+        fieldGroupElement.append(additionalElement).append(fieldElement);
+
+      });
+
+      return fieldGroupElement;
+    };
+
+    self.createHeaderElement = function(selectedAsset) {
+      var title = function () {
+        var asset = selectedAsset.getCurrentLane();
+
+        return asset.id !== 0 && !selectedAsset.isSplit() ?
+            '<span>Kohteen ID: ' + asset.id + '</span>' :
+            '<span>' + self._assetTypeConfiguration.title + '</span>';
+      };
+
+      return $(title());
+    };
+
+    var createPreviewHeaderElement = function(laneNumbers) {
+      var createNumber = function (number) {
+        var previewButton = $('<td class="preview-lane">' + number + '</td>').click(function() {
+          $(".preview .lane").removeClass("not-highlight-lane highlight-lane").addClass("not-highlight-lane");
+          $(this).removeClass("not-highlight-lane").addClass("highlight-lane");
+          var laneNumber = parseInt(number);
+          lanesAssets.setCurrentLane(laneNumber);
+
+          if(laneNumber.toString()[1] == "1"){
+            currentFormStructure = mainLaneFormStructure;
+          }else{
+            defaultFormStructure.fields[0] = {label: 'Kaista', type: 'read_only_number', publicId: "lane_code", defaultValue: laneNumber, weight: 6};
+            currentFormStructure = defaultFormStructure;
+          }
+          reloadForm($('#feature-attributes'));
+        });
+
+        if(number == lanesAssets.getCurrentLaneNumber()){
+          return previewButton.addClass("highlight-lane");
+        }else{
+          return previewButton.addClass("not-highlight-lane");
+        }
+      };
+
+      var numbers = _.sortBy(laneNumbers);
+
+      var odd = _.filter(numbers, function (number) {
+        return number % 2 !== 0;
+      });
+      var even = _.filter(numbers, function (number) {
+        return number % 2 === 0;
+      });
+
+      var preview = function () {
+        var previewList = $('<table class="preview">');
+
+        var numberHeaders =$('<tr style="font-size: 10px;">').append(_.map(_.reverse(even).concat(odd), function (number) {
```

Comment:
Things like below it's to be put at CSS files, please.
```javascript
'<tr style="font-size: 10px;">'

Reply:
Done